.travis/scripts/prepare_release.sh

```diff
@@ -2,6 +2,20 @@
 set -ev
 
 # Prepare release
+LATEST_RELEASE_TAG=$(git describe --abbrev=0)
```

Comment:
I would better call the variable PREVIOUS_RELEASE_TAG

Reply:
Done